save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -18,9 +18,9 @@ abstract class Plugin(open val testConfig: TestConfig, private val testFiles: Li
      *
      * @return a sequence of [TestResult]s for each group of test resources
      */
-    fun execute(): Sequence<TestResult> = handleFiles(
+    open fun execute(): Sequence<TestResult> = handleFiles(
```

Comment:
Between making this method open and override it in every plugin and not passing `regex` via argument into `rawDiscoverTestFiles`, I'd choose the latter. You can take regex from plugin config in every plugin.